checkov/secrets/runner.py

```diff
@@ -127,7 +133,13 @@ def run(
                     filter_ignored_paths(root, d_names, excluded_paths)
                     filter_ignored_paths(root, f_names, excluded_paths)
                     for file in f_names:
-                        if file not in PROHIBITED_FILES and f".{file.split('.')[-1]}" in SUPPORTED_FILE_EXTENSIONS:
+                        if runner_filter.secret_scan_extension_by_file_type:
+                            if file == 'Dockerfile' or f".{file.split('.')[-1]}" in ADDED_TO_SECRET_SCAN_FILES_TYPES or f".{file.split('.')[-1]}" in SUPPORTED_FILE_EXTENSIONS:
```

Comment:
as mentioned before `file == 'Dockerfile'` should be against our Dockerfile regex